workflow_ppk.py

```diff
@@ -350,8 +357,12 @@ def main(datadir, geoid, makePos=True, verbose=2, sonar_method='default', rtklib
                     elevation_out[tidx] = T_ppk['GNSS_elevation_NAVD88'][idxTimeMatchGNSS] - antenna_offset - \
                                           sonarData['this_ping_depth_m'][idxTimeMatchSonar]
                     sonar_out[tidx] = sonarData['this_ping_depth_m'][idxTimeMatchSonar]
+                elif sonar_method == 'qaqc':
+                    elevation_out[tidx] = T_ppk['GNSS_elevation_NAVD88'][idxTimeMatchGNSS] - antenna_offset - \
+                                          sonarData['qaqc_depth_m'][idxTimeMatchSonar]
```

Comment:
i don't love this, but it'll work for this PR.  I think my branch has the variable assigned as `this_ping_depth_m` and overwrites the oem line. ... i don't know that i love that either.